test/powershell/Modules/Microsoft.PowerShell.Utility/ConvertTo-Json.Tests.ps1

```diff
@@ -40,6 +40,11 @@ Describe 'ConvertTo-Json' -tags "CI" {
         $ps.Dispose()
     }
 
+    It "Should accept minimum depth as 0." {
+        { $script:output = 1 | ConvertTo-Json -Depth 0 } | Should -Not -Throw
+        $script:output | Should -Be 1
+    }
```

Comment:
```suggestion
    It "Should accept minimum depth as 0." {
        $output = 1 | ConvertTo-Json -Depth 0
        $output | Should -Be 1
    }
```

Reply:
I'd prefer more complex object instead of 1 so that we see truncation.

Reply:
The first check was to ensure the acceptance of Depth 0. Do we need to remove it ?

Reply:
Yes, now the cmdlet doesn't throw in the case and we need a _positive_ test.

Reply:
Please use more complex source object in the test so that we get something like "AnotherObject":"System.Collections.Hashtable" as in examples above.

Reply:
Please address my last comment.

Reply:
Done